frontend/src/components/PhoneNumberInput/IntlPhoneNumberInput.stories.tsx

```diff
@@ -92,7 +92,9 @@ export const Playground: Story = ({
             required: isRequired
               ? { value: true, message: 'Required field' }
               : false,
-            validate: (val) => isValidPhoneNumber(val) || 'Invalid number',
+            validate: (val) => {
+              return isValidPhoneNumber(val) || 'Invalid number'
```

Comment:
@timotheeg here is the relevant line; however note that this is a story-only function and not really part of the component, and can be replaced with another validation function depending on the caller

Reply:
Roger that. I spotted the import for mobile-only verification:
```typescript
import { isValidPhoneNumber } from 'libphonenumber-js/mobile'
```

All good 👍